coursedata/level-defaults/nb_NO.yaml

```diff
@@ -1,354 +1,354 @@
+    start_code: "navn = 'Hedy'\n{print}('Mitt navn er ', navn)"
+    intro_text: "## Level 18\nVi har ankommet nivået hvor vi skriver virkelig pythonkode! Python er riktignok ikke på norsk, men du kan bruke knappen oppe i høyre hjørne av kodefeltet hvis du vil se hvordan koden ser ut på engelsk.\nNå når vi har nådd ekte pythonkode må vi bruke paranteser når vi bruker {print} og {range}.\nDette ser slik ut:\n```\n{print}('Mitt navn er Hedy!')\n```\nFor å skrive ut mer enn en ting må du bruke et komma mellom tingene.\n```\nnavn = 'Hedy'\n{print}('Mitt navn er ', navn)\n```"
     commands:
     -   name: '{print}'
-        explanation: With `{print}` you need to use parentheses
+        explanation: Med`{print}` må du bruke paranteser
         example: '{print}'
-        demo_code: "{print}('hi!')"
-    -   name: '{print} with var'
-        explanation: With `{print}` you need to use parentheses and commas if you {print} more items
+        demo_code: "{print}('hei!')"
+    -   name: '{print} med var'
+        explanation: Med `{print}` må vi bruke parantes og komma for å skrive ut flere ting
         example: '{print}'
-        demo_code: "naam = 'Hedy'\n{print}('my name is ', naam)"
+        demo_code: "navn = 'Hedy'\n{print}('Mitt navn er ', navn)"
 17:
-    start_code: "{for} i {in} {range} 1 {to} 10:\n    {print} i\n{print} 'Ready or not, here I come!'"
-    intro_text: "Now we are going to change a little bit with indentation. Every time that we need an indentation, we need `:` at the line before the indentation.\n\nIn this level you can also use a new command: `{elif}`. `{elif}` is short for {else} {if} and you need it when you you want to make 3 (or more!) options.\nCheck it out!\n```\nprices = ['1 million dollars', 'an apple pie', 'nothing']\nyour_price = prices[random]\n{print} 'You win ' your_price\n{if} your_price == '1 million dollars' :\n    {print} 'Yeah! You are rich!'\n{elif} your_price == 'an apple pie' :\n    {print} 'Lovely, an apple pie!'\n{else}:\n    {print} 'Better luck next time..'\n```\n"
+    start_code: "{for} i {in} {range} 1 {to} 10:\n    {print} i\n{print} 'Klar eller ei, her kommer jeg!'"
+    intro_text: "Nå gjør vi en liten endring i koden når vi bruker indentering. Hver gang vi trenger indentering, må vi nå skrive `:` på linjen før indenteringen.\n\nPå dette nivået kan du også bruke en ny kommando `{elif}`. `{elif}` er en forkortelse for {else} {if} og du trenger det når du har lyst til å lage kode med 3 (eller fler!) alternativer.\nSjekk det ut!\n```\npremier = ['1 million kroner', 'en eplekake', 'ingenting']\ndin_premie = premie[{random}]\n{print} 'Du vinner ' din_premie \n{if} din_premie == '1 million kroner':\n    {print} 'Hurra! Du er rik!'\n{elif} din_premie == 'en eplekake':\n    {print} 'Digg, eplekake!'\n{else}:\n    {print} 'Bedre lykke neste gang..'\n```\n"
```

Comment:
This fails as well! Because we first create a list called `premier` and then try to select a random value from this list by calling `premie`. As `premie` doesn't exist as an variable it results in an error!

Reply:
Fixed in weblate as well!

Reply:
I even translated the error message for this, and common mistakes section in for_teachers, my fault 😅